soc/spid.c

```diff
@@ -108,8 +108,7 @@ struct pollfd fds[N_POLLFDS];
 int usbd_sock_fd;
 struct sockaddr_un usbd_sock_addr;
 void delay() {
-    volatile int i = 1000;
-    while(i--);
```

Comment:
You previously [had concerns about using usleep `here`](https://github.com/tessel/t2-firmware/commit/df1da92cd7ecd9b41dd66a754caf9780ee8ba7e5) - is that not an issue anymore?